charts/emissary-ingress/templates/_helpers.tpl

```diff
@@ -124,24 +124,4 @@ Define the http port of the Ambassador service
 {{ default .port }}
 {{- end -}}
 {{- end -}}
-{{- end -}}
-
-{{/*
-Set the image that should be used for the agent.
-Use agent.repository and agent.tag if present,
-Then use fullImageOverride if present,
-Then if the image repository is explicitly set, use "repository:image"
-*/}}
-{{- define "agent.image" -}}
-
-{{- if .Values.agent.image.repository -}}
-    {{- printf "%s:%s" .Values.agent.image.repository .Values.agent.image.tag -}}
-{{- else if .Values.image.fullImageOverride -}}
-    {{- .Values.image.fullImageOverride }}
-{{- else if hasKey .Values.image "repository" -}}
-    {{- printf "%s:%s" .Values.image.repository .Values.image.tag -}}
-{{- else -}}
-    {{- printf "%s:%s" "docker.io/emissaryingress/emissary" .Values.image.tag -}}
-{{- end -}}
-
-{{- end -}}
+{{- end -}}
```

Comment:
nit: When you see this little Icon at the bottom of a file in the github changes view it means that there should be an empty new line at the end of the file. Try not to erase those on files you edit.

Reply:
If you use vscode then you can go look for the extension `EditorConfig.EditorConfig` and it will respect the contents of the `.editorconfig` file and do that automatically for you when you save. AFAIK most other editors can be made to respect `.editorconfig` as well, otherwise just follow the established patterns you see.